assets-src/styles/sass/30-base/_lists.scss

```diff
@@ -32,11 +22,25 @@ ol.counter {
 }
 
 
-// No bullets or numbering
+/**
+Remove bullets, margin and padding from lists inside a <nav> element, and lists with the
+'.clean-list' class/. Note that that this may remove list semantics in some circumstances in
+some WebKit browsers. See https://www.scottohara.me/blog/2019/01/12/lists-and-safari.html
+If it's important that the number of items in a list is announced, add role="list" to the <ul> / <ol>
+ */
+nav ul,
+nav ol,
 .clean-list {
-	li {
-		display: block;
-	}
+    list-style: none;
+    margin: 0;
+    padding: 0;
+
+    li,
+    ol,
+    ul {
+        margin: 0;
+        padding: 0;
```

Comment:
We could, however it would reduce the target size height of the links from 44px to 36px so they would no longer meet [WCAG 2.2 SC 2.5.5 Target Size (Enhanced) (Level AAA)](https://www.w3.org/WAI/WCAG22/Understanding/target-size-enhanced.html), which would be a shame.

Reply:
What's strange is that the TOC items on that page have an even smaller height (22.75px)... 
The TOC matches the [example 3](https://www.w3.org/WAI/WCAG22/Understanding/target-size-enhanced.html#examples) listed on that page so my understanding is that this is acceptable to have smaller height for that specific case.

Reply:
When I inspect the Page contents menu in Chrome on the linked WCAG Understanding Docs page, although the list items have a height of 22.75px (even smaller for the first list item) the links themselves have a height of 24px. So that's AA compliant.

> The TOC matches the [example 3](https://www.w3.org/WAI/WCAG22/Understanding/target-size-enhanced.html#examples) listed on that page so my understanding is that this is acceptable to have smaller height for that specific case.

Ah - I was going by [example 5](https://www.w3.org/WAI/WCAG22/Understanding/target-size-enhanced.html#examples) on that page!

Reply:
Sorry, I'm a bit confused. I see the links in the TOC on [WCAG 2.2 SC 2.5.5 Target Size (Enhanced) (Level AAA)](https://www.w3.org/WAI/WCAG22/Understanding/target-size-enhanced.html) have a height of 14px which is far from the 44px recommended in that document.

Reply:
Huh - here's a screenshot of what I see in my Chrome browser:

![Screenshot of Understanding docs page as viewed in Google Chrome, with developer tools open and link highlighted to show dimensions](https://github.com/user-attachments/assets/dd073221-41f2-4842-b51e-9eb43b4f1c36)

It's a little off topic to be fair - the design for the page is not meeting the AAA requirement and that's somebody's decision, but it does spell out what the requirements are if people do want to meet the AAA requirement.

I have pushed a change - scoped to the `.toc`class on the parent - to reduce the line height as suggested. The height of the link targets will still exceed AA but will no longer meet AAA.

Reply:
Thanks @NicolaSaunders